pkg/okteto/preview_test.go

```diff
@@ -166,14 +167,48 @@ func TestDeployPreview(t *testing.T) {
 				err: nil,
 			},
 		},
+		{
```

Comment:
did you consider checking if labels are returned?

Reply:
We will when we list the previews